src/content/api/introduction.mdx

```diff
@@ -1,8 +1,9 @@
 ---
-title: API Introduction
+title: Introduction
 description: Introduction to the API Reference
+draft: true
```

Comment:
@AndyClifford was this meant to be on the existing page you made? I can't see where it was getting set 😅